yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/launcher/DockerComputerSSHLauncher.java

```diff
@@ -114,11 +117,16 @@ public HostAndPort getHostAndPort(String cloudId, InspectContainerResponse ir) {
         String host = null;
         Integer port = 22;
 
-        final InspectContainerResponse.NetworkSettings networkSettings = ir.getNetworkSettings();
+        final NetworkSettings networkSettings = ir.getNetworkSettings();
         final Ports ports = networkSettings.getPorts();
         final Map<ExposedPort, Ports.Binding[]> bindings = ports.getBindings();
         final Ports.Binding[] sshBindings = bindings.get(sshPort);
 
+        if (isNull(sshBindings)) {
+            throw new IllegalStateException("SSH Binding not found for " + ir.getId() +
```

Comment:
@dasj i also getting such errors, so i enhanced error message here. PR would be merged later.